### PR TITLE
Adds animation and external js libraries that make the page more interactive 

### PR DESCRIPTION
**Pull request by Aradhya-B**
image

## Libraries Added
- lightbox.js: used to overlay images on current page (https://lokeshdhakar.com/projects/lightbox2/)
- baffle.js: obfuscates and reveals text in DOM elements (https://camwiegert.github.io/baffle/)

### LightBox
image

### Baffle
image

Code is still messy in this PR because it's retroactive, but formatting and style changes you suggested in the last PR and this one will be reflected in the third and final PR.

### CODE REVIEW

File: portfolio/src/main/webapp/images/icons/mail.svg [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (nickKelman) [inline portfolio/src/main/webapp/images/icons/mail.svg:42]: Can these be deleted?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -29,114 +37,123 @@ <h5 class="subtitle">
...
-          <li>
-            '14-'15: Lost over 50 pounds and became a starter for the same team
+            I was born and raised in Calgary, Alberta
           </li>
           <li>
-            '16-'17: Became a starter for my high school and AAU basketball
-            teams
+            I have an 
+            <a class="list-link" 
+              href="images/family.png" data-lightbox="Family" data-title"Family">
```
Comment (nickKelman) [inline portfolio/src/main/webapp/index.html:47]: missing =

Comment (nickKelman) [reply portfolio/src/main/webapp/index.html:47]: Applies to other data-title's in this file

**Review by nickKelman — APPROVED** [review]

File: portfolio/src/main/webapp/images/icons/mail.svg (hunk 1 above)
Comment (Aradhya-B) [reply portfolio/src/main/webapp/images/icons/mail.svg:42]: Yep, idk why it automatically added all the extra tags. Will be in next pr.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (Aradhya-B) [reply portfolio/src/main/webapp/index.html:47]: Got it.

**Pull request merged**